File: test/fixtures/python/multiple_urls.py
import requests

data = {
    'fooo': 'blah',
}

with open('file.txt', 'rb') as f:
    file_contents = f.read()
response = requests.put('http://localhost:28139/file.txt', data=file_contents)

params = {
    'params': 'perurltoo',
}
with open('myfile.jpg', 'rb') as f:
    file_contents = f.read()
response = requests.put('http://localhost:28139/myfile.jpg', params=params, data=file_contents)

response = requests.post('http://localhost:28139', data=data)


data = {
    'different': 'data',
    'time': 'now',
}

response = requests.post('http://localhost:28139', data=data)

File: test/fixtures/node/multiple_urls.js
import fetch, { fileFromSync } from 'node-fetch';

fetch('http://localhost:28139/file.txt', {
  method: 'PUT',
  body: fileFromSync('file.txt')
});
fetch('http://localhost:28139/myfile.jpg?params=perurltoo', {
  method: 'PUT',
  body: fileFromSync('myfile.jpg')
});
fetch('http://localhost:28139', {
  method: 'POST',
  body: new URLSearchParams({
    'fooo': 'blah'
  })
});

fetch('http://localhost:28139', {
  method: 'POST',
  body: new URLSearchParams({
    'different': 'data',
    'time': 'now'
  })
});
